docs/release-notes/2.28.0.md

```diff
@@ -0,0 +1,105 @@
+# Percona Monitoring and Management 2.28.0
+
+<table class="docutils field-list" frame="void" rules="none">
+  <colgroup>
+    <col class="field-name">
+    <col class="field-body">
+  </colgroup>
+  <tbody valign="top">
+    <tr class="field-odd field">
+    <th class="field-name">Date:</th>
+    <td class="field-body">May 12, 2022</td>
+    </tr>
+    <tr class="field-even field">
+      <th class="field-name">Installation:</th>
+      <td class="field-body">
+        <a class="reference external" href="https://www.percona.com/software/pmm/quickstart">Installing Percona Monitoring and Management</a></td>
+    </tr>
+  </tbody>
+</table>
```

Comment:
This whole table is somewhat redundant, since we can achieve the same goal with just a few lines of markdown. Do you want me to try?

Reply:
And the formatting is a bit broken
<img width="781" alt="Screenshot 2022-05-05 at 13 08 44" src="https://user-images.githubusercontent.com/81549/166903183-c533dba3-6f56-4ea2-abb6-08d5ebc7f14d.png">

Reply:
Sure!